addons/addon-base-raas/packages/base-raas-services/lib/schema/create-aws-accounts.json

```diff
@@ -1,45 +1,22 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "type": "object",
-  "additionalProperties": false,
-  "properties": {
-    "name": {
-      "type": "string",
-      "minLength": 1,
-      "maxLength": 100
-    },
-    "description": {
-      "type": "string",
-      "maxLength": 2048
-    },
-    "accountId": {
-      "type": "string",
-      "pattern": "^[0-9]{12}$"
-    },
-    "roleArn": {
-      "type": "string",
-      "minLength": 10
-    },
-    "xAccEnvMgmtRoleArn": {
-      "type": "string",
-      "minLength": 10
-    },
-    "externalId": {
-      "type": "string",
-      "minLength": 1
-    },
-    "vpcId": {
-      "type": "string",
-      "pattern": "^vpc-[a-f0-9]{8,17}$"
-    },
-    "subnetId": {
-      "type": "string",
-      "pattern": "^subnet-[a-f0-9]{8,17}$"
-    },
-    "encryptionKeyArn": {
-      "type": "string",
-      "pattern": "^arn:aws:kms:.*$"
-    }
-  },
-  "required": ["name", "roleArn", "externalId", "accountId", "vpcId", "subnetId", "encryptionKeyArn"]
-}
```

Comment:
This might break the `Create Account` functionality. Could you please do a global search on the createAccount method and make sure it's not breaking anything?

Reply:
sure, I'll look into that today

Reply:
it doesn't look like it's breaking any functionality. `Create Account` uses a completely separate service, store, and schema for its stuff, so this schema is only used for the add account feature. The Create Account functionality actually just adds the information directly to the backend db, so it doesn't reference the `aws-accounts-service` at all

Reply:
Gotcha. That's interesting. Thanks for looking into this :)